src/cascadia/TerminalSettingsModel/MTSMSettings.h

```diff
@@ -57,7 +57,8 @@ Author(s):
     X(bool, AlwaysShowNotificationIcon, "alwaysShowNotificationIcon", false)                                                                               \
     X(winrt::Windows::Foundation::Collections::IVector<winrt::hstring>, DisabledProfileSources, "disabledProfileSources", nullptr)                         \
     X(bool, ShowAdminShield, "showAdminShield", true)                                                                                                      \
-    X(bool, TrimPaste, "trimPaste", true)
+    X(bool, TrimPaste, "trimPaste", true)                                                                                                                  \
+    X(Model::ConsoleHostStartupMode, ConsoleHostStartupMode, "consoleHostStartupMode", ConsoleHostStartupMode::Default)

```

Comment:
We should rename this variable/setting, at the very minimum to get away from the term "consoleHost". It seems like this setting is intended to have a description kind of like this:
`"determines the launch mode when the terminal is handed off (only applicable if defterm is enabled)"`

That said, how about a name like `handoffLaunchMode` or `deftermLaunchMode`?